### DISCUSSION TITLE
Motors not working on Speedybee f405 v3

### DISCUSSION THREAD

**Discussion by RGBTyp** [iNavFlight/inav#9053] [General]
I tried installing inav 6.0 on this flight Controller. 

The first thing i tried is setting up the motor outputs. No matter what protocoll i choose none of them will spin. 
I have already set up another quad with inav so I am not completly new but i must have forgotten something because else i would not be the only one with this issiue. 
The motors work just fine in betaflight. 
Is there anything else that I need to do other than enabeling motor output and choosing the protocoll?
I have changed no other settings.

**Comment by Demoskin** [comment]
Motors working but, with blheli_s only. With Blujay didn't start. Biderection not working too, Protocol Multishot and Oneshot125 only. Dshot300/600 not supported.

**Comment by Trankilo0255** [comment]
> I tried installing inav 6.0 on this flight Controller.
> 
> The first thing i tried is setting up the motor outputs. No matter what protocoll i choose none of them will spin. I have already set up another quad with inav so I am not completly new but i must have forgotten something because else i would not be the only one with this issiue. The motors work just fine in betaflight. Is there anything else that I need to do other than enabeling motor output and choosing the protocoll? I have changed no other settings.

I have exactly the same issue...didi you manage to solve it in the mean time?

**Reply by Jetrell** [reply]
> I have exactly the same issue...did you manage to solve it in the mean time?

SpeedyBee didn't check in the design process, if the outputs, S1 to S4 DMA timers are all defined for motor use on the F405 in INAV.
So it has NO Dshot compatibility.

**Reply by MusafirFpv** [reply]
try this 
https://youtu.be/9B6sel9jCF0?si=J9hegkAiRb58P-aw

**Comment by MusafirFpv** [comment]
try this for esc calibration
 
https://youtu.be/9B6sel9jCF0?si=J9hegkAiRb58P-aw

**Comment by 101Garfield** [comment]
Had the problem that the motors were not spinning after flashing Inav (did work out of the box with BetaFlight)
Found this site: https://speedybee.zendesk.com/hc/en-us/articles/13944314528283-How-does-F405v3-FC-use-servos-in-INAV
It worked after i used  "load Mixer".... that is motors were spinning in "outputs:"
now they don't work, probably because of low battery
